contributing_to_docs/doc_guidelines.adoc

```diff
@@ -693,36 +703,31 @@ To provide an example URL path that you do not want to render as a hyperlink, us
 ....
 
 === Internal cross-references
-Whenever possible the link to another assembly should be part of the actual sentence.
-Avoid creating links as a separate sentence that begins with "See [this assembly] for more information on x".
 
-[NOTE]
-====
-Use the relative file path (from the file you are editing, to the file you are linking to),
-even if you are linking to the same directory that you are writing in. This makes search and replace
-operations to fix broken links much easier.
+Use the relative file path (from the file you are editing to the file you are linking to), even if you are linking to the same directory that you are writing in. This makes search and replace operations to fix broken links much easier.
 
-For example, if you are writing in *_architecture/core_concepts/deployments.adoc_* and you want to
-link to *_architecture/core_concepts/routes.adoc_* then you must include the path back to the first
-level of the assembly directory:
+For example, if you are writing in *_architecture/core_concepts/deployments.adoc_* and you want to link to *_architecture/core_concepts/routes.adoc_*, then you must include the path back to the first level of the assembly directory:
```

Comment:
Ah yea, this was our (very) old markup rule for file names. I was mostly leaving old markup in this document as-is because it's so all over the place in the exposition, but I will sync this up. :)